www/runtest.php

```diff
@@ -2864,36 +2873,6 @@ function GetClosestLocation($url, $browser) {
     return $location;
 }
 
-function ErrorPage($error) {
-    global $privateInstall;
-    global $supportsAuth;
-    global $supportsSaml;
-    global $USER_EMAIL;

```

Comment:
`$USER_EMAIL`, `$supportsAuth` and `$supportsSaml` all get used by the header, so we need to get this information into the template somehow. (I don't see footer referencing any of these, nor do I see `$privateInstall` used at all here)

These all feel like the kind of global information we may just want to have readily available to templates throughout in some way. Does it make sense for the template class to magically pull in some basic global data or is it better to be explicit about this?